src/GuiBase/Viewer/TrackballCamera.hpp

```diff
@@ -26,6 +26,11 @@ class TrackballCamera : public CameraInterface {
     /// @return true if the event has been taken into account, false otherwise
     virtual bool handleKeyReleaseEvent( QKeyEvent* event ) override;
 
+    void setCamera( Engine::Camera* camera ) override;
+
+    void save( std::ostream& out ) const override;
+    void load( std::istream& in ) override;
+

```

Comment:
Not sure these method must be defined. Why if we want to save the camera in another format than the .cam fil format ? Again, let direct load/save of a camera be part of a dedicated plugin.